open_rarity/scoring/scorer.py

```diff
@@ -42,24 +62,63 @@ def score_tokens(
             collection (Collection): The collection to score from
             tokens (list[Token]): a batch of tokens belonging to collection
                 to be scored
-            normalized (bool, optional): _description_. Defaults to True.
+            normalized (bool, optional):
+                Set to true to enable individual trait normalizations based on
+                total number of possible values for an attribute name.
+                Defaults to True.
 
         Returns:
             list[Score]: list of scores in order of `tokens`
         """
         self.validate_collection(collection=collection)
+        return self.handler.score_tokens(
+            collection=collection, tokens=tokens, normalized=normalized
+        )
 
     def score_collection(
         self, collection: Collection, normalized: bool = True
-    ) -> list[Score]:
+    ) -> list[float]:
         """Scores all tokens on collection.tokens
 
+        Args:
```

Comment:
Numpy doc

Reply:
Updating all of Scorer and InformationContent to use numpy doc!

Reply:
And scoringhandler